pkg/types/rfc3161/v0.0.1/rfc3161_v0_0_1_schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/timestamp/timestamp_v0_0_1_schema.json",
+    "title": "Timestamp v0.0.1 Schema",
+    "description": "Schema for RFC3161 entries",
+    "type": "object",
+    "properties": {
+        "tsr": {
+            "description": "Information about the tsr file associated with the entry",
+            "type": "object",
+            "properties" : {
+                "content": {
```

Comment:
just curious - why is the hash not included in the `tsr` object (as in the other types)? if we don't end up persisting the content of the TSR into the tlog, including the hash ensures we don't run into collisions when we insert multiple TSRs signed with the same public key.

Reply:
no particular reason, I had it in there, but it didn't seem to add any value? I though the TSR content should be enough to differentiate between the entries? Is that not true?

Reply:
If we don't store the content (given precedent from other types) then you'd just have the public key left here which runs into the issues of potential collisions for a TSA signing two or more timestamps which would appear to the tlog as duplicate entries. Since we're not storing a separate signature value in the type, we need something to provide uniqueness.

Reply:
if you were to opt to store a hash, i would either store a hash of the entire TSR or just the signed content like I mentioned below https://github.com/sigstore/rekor/pull/324#discussion_r649953085

both should provide uniqueness, even across the same TSA, same content, same time (TSA creates a serial number included in the signed content that shouldn't be repeated).

Reply:
Using hash of the entire file